README.md

```diff
@@ -50,5 +50,5 @@ DESCRIPTION
   us know here: https://twil.io/twilio-cli-feedback
 ```
 
-_See code: [src/commands/debugger/logs/list.js](https://github.com/twilio/plugin-debugger/blob/v1.1.9/src/commands/debugger/logs/list.js)_
+_See code: [src/commands/debugger/logs/list.js](https://github.com/twilio/plugin-debugger/blob/master/src/commands/debugger/logs/list.js)_
```

Comment:
Should just be `/1.1.9/` since we don't use the `v` prefix: https://github.com/oclif/dev-cli/blob/v1.22.2/src/commands/readme.ts#L170

Can add this to the `oclif` section in `package.json` to fix it going forward (removed the `v`):
```
"repositoryPrefix": "<%- repo %>/blob/<%- version %>/<%- commandPath %>"
```

Reply:
👍 thanks! will make those changes shortly!